queries.md

```diff
@@ -2,22 +2,52 @@
 
 ## find all customers that live in London. Returns 6 records.
 
+SELECT * FROM [Customers] WHERE City = 'London'
+ORDER BY CustomerID
+
 ## find all customers with postal code 1010. Returns 3 customers.
 
+SELECT * FROM [Customers] WHERE PostalCode LIKE '1010%'
+
 ## find the phone number for the supplier with the id 11. Should be (010) 9984510.
 
+SELECT Phone FROM Suppliers WHERE SupplierID = 11
+
 ## list orders descending by the order date. The order with date 1997-02-12 should be at the top.
 
+SELECT * FROM [Orders] ORDER BY OrderDate DESC
+
 ## find all suppliers who have names longer than 20 characters. You can use `length(SupplierName)` to get the length of the name. Returns 11 records.
 
+SELECT * FROM [Suppliers] WHERE length(SupplierName) > 20
+ORDER BY SupplierName
+
 ## find all customers that include the word "market" in the name. Should return 4 records.
 
+SELECT * FROM [Customers] WHERE CustomerName LIKE '%market%'
+
 ## add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_.
 
+INSERT INTO
+Customers(CustomerName, ContactName, Address, City, PostalCode, Country)
+VALUES ("The Shire", "Bilbo Baggins", "1 Hobbit-Hole", "Bag End", "111", "Middle Earth")
+
+92	The Shire	Bilbo Baggins	1 Hobbit-Hole	Bag End	111	Middle Earth
+
 ## update _Bilbo Baggins_ record so that the postal code changes to _"11122"_.
 
+UPDATE Customers
+SET PostalCode = 11122
+WHERE CustomerID = 92
+
+92	The Shire	Bilbo Baggins	1 Hobbit-Hole	Bag End	11122	Middle Earth
+
+
 ## list orders grouped by customer showing the number of orders per customer. _Rattlesnake Canyon Grocery_ should have 7 orders.
 
+SELECT CustomerName, Count(Orders.CustomerID) FROM Customers
+INNER JOIN Orders ON Customers.CustomerID = Orders.CustomerID GROUP BY Customers.CustomerID ORDER BY Count(Orders.CustomerID) DESC
```

Comment:
Good job using multiple lines to write queries. This makes it much easier to read, especially when queries get longer. With this query, I would suggest breaking it down even more to avoid lines longer than 80 characters. Here is a great article about SQL query structure(halfway down the page): https://solutioncenter.apexsql.com/sql-formatting-standards-joins-lists-structure-operations/